FileReeder.java

```diff
@@ -0,0 +1,33 @@
+import java.io.File;
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+
+public class FileReeder {
```

Comment:
What is this name? Generally, spelling matters. Also, names should be clear. This class does not really read files. It does searches in files. Perhaps name is "search" or "searcher" or similar.